content/en/seameet/seameet-manual/13-seameet-google-doc.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Export Meeting to Google Doc"
+description: ""
+date: 2023-11-22T08:48:57+00:00
+lastmod: 2024-06-03T08:48:57+00:00
+draft: false
+images: []
+menu:
+  seameet:
+     parent: "seameet-manual"
+aliases:
+   - /en/seameet/13-seameet-google-doc/
+weight: 113
+toc: true
+---
+
+After the meeting ends, the meeting record will be exported as a Google Docs document. Click the button in the top right corner of the page to open Google Docs, which contains all the information from the just-concluded meeting.
+
+1. Bind Google Drive
+
+   If you want to export the meeting record as a Google document, please bind the workspace to a Google Drive. You can bind Google Drive in **Workspace** - **Integration**.
+
+    <center>
+
+    <img src="/images/seameet-en/13-seameet-google-doc/seameet-bind-google-drive.png" alt="SeaMeet Bind Google Drive"/>
+
+    </center>
+
+2. Automatically Export Google Document After Meeting
+
+   Click the Google Docs button in the top right corner of the meeting page to access the corresponding meeting record
+
+    <center>
+
+    <img src="/images/seameet-en/13-seameet-google-doc/seameet-auto-export-google-doc.png" alt="SeaMeet Auto Export Google Document"/>
+
+    </center>
```

Comment:
1. do you have a screenshot of the google doc?
2. explain that the exported docs live in user's google drive and doesn't belong to seasalt.ai

Reply:
I put a screenshot of the downloaded screenshots. Please have a look and see if it is what we want.